### PR TITLE
[ENH] added md files from bids-starter-kit repository

### PR DESCRIPTION
**Pull request by robertoostenveld**
I believe that all menu items under "Getting started" in the new navigation now have their corresponding page.

### CODE REVIEW

**Comment by robertoostenveld** [comment]
@Remi-Gau can you review and merge in the `enh/new_nav` branch that you have created?

File: docs/collaboration/code_of_conduct.md
Comment (Remi-Gau) [inline docs/collaboration/code_of_conduct.md:1]: right now we are using the code of conduct of the spec: we have an issue (and a future working group) to uniformize our code of conduct across repos.

for this reason I would not move the code of conduct in the website.

Comment (Remi-Gau) [inline docs/collaboration/code_of_conduct.md:1]: right now we are using the code of conduct of the spec: we have an issue (and a future working group) to uniformize our code of conduct across repos.

for this reason I would not move the code of conduct in the website.

File: docs/getting_started/src
Comment (Remi-Gau) [inline docs/getting_started/src:1]: Can you open a PR on the `bfm` branch of the starter kit to explain where each page has been moved ?

See for example:
https://github.com/bids-standard/bids-starter-kit/blob/bfm/src/glossary.md?plain=1

**Review by Remi-Gau — CHANGES_REQUESTED** [review]

**Comment by Remi-Gau** [comment]
I can take care of those changes if you want Robert.

**Comment by Remi-Gau** [comment]
will push additional commits on this branch

**Review by Remi-Gau — APPROVED** [review]

**Comment by Remi-Gau** [comment]
will fix links later

**Pull request merged**